Tools/LyTestTools/ly_test_tools/cli/find_cpp_library.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Tools for inspecting GitHub CODEOWNERS files
+"""
+
+import os
+
+from ly_test_tools.cli.codeowners_hint import get_codeowners
+
+
+def find_cpp_library_from_test_module_name(folder_to_search, test_module_name):
+    """
+    This function searches through the CMakeLists.txt in a given folder for what library added the given module.
+    :param folder_to_search: the folder location to start the search at, recursively
+    :param test_module_name: the name of the test module
+    :return: the full file path to the library
+    """
+    cpp_library_full_path = ""
+    print(f"Starting search in {folder_to_search}")
+    for root, subdirs, files in os.walk(folder_to_search):
+        # print(f"walking {folder_to_search}")
+        for filename in files:
+            if filename == "CMakeLists.txt":
```

Comment:
Instead of iterating, should be faster to just check for `if os.path.exists(os.path.join(folder_to_search, 'CMakeLists.txt'))`